pkg/squashfs/squashfs.go

```diff
@@ -198,11 +208,44 @@ func isMountedAtDir(src, dest string) (bool, error) {
 	return false, nil
 }
 
-func findSquashfusePath() string {
+func findSquashFuseInfo() {
+	var sqfsPath string
 	if p := which("squashfuse_ll"); p != "" {
-		return p
+		sqfsPath = p
+	} else {
+		sqfsPath = which("squashfuse")
 	}
-	return which("squashfuse")
+	if sqfsPath == "" {
+		return
+	}
+	version, supportsNotfiy := sqfuseSupportsMountNotification(sqfsPath)
+	squashFuseInfo = squashFuseInfoStruct{sqfsPath, version, supportsNotfiy}
```

Comment:
I think it would be good to have a Info level log here that says something like:

log.Infof("Found squashfuse at %s (version=%s notify=%t)", squashFuseInfo.Path, squashFuseInfo.version, squashFuseInfo.supportsNotify)